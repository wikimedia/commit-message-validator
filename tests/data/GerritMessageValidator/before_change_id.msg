This is the subject line of the commit

Change-Id: Ifcd397165df1cbf9fa04f2044e1bb33ad7414d8d
Bug: T142804
