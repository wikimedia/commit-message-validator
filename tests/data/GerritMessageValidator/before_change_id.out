The following errors were found:
- Line 4: Expected 'Bug:' to come before Change-Id on line 3
